ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -100,6 +100,11 @@ public static string ValidYearLaterThanBirthYear(string name, int birthYear)
         public const string ImmunosuppressionTypeRequired = "At least one field must be selected";
         #endregion
 
+        #region Social Context
+        public const string VenueDateShouldBeLaterThanDob = "Must be later than date of birth";
```

Comment:
I think we want to include the name of the field for the purposes of correct display in error summaries, perhaps using the `{0}` syntax - I imagine this comment will make more sense once merged with master changes!
There might be some other bits around the error summaries that will need to be added!

Reply:
In fact, for this particular message you could reuse https://github.com/PublicHealthEngland/ntbs_Beta/blob/master/ntbs-service/Models/Validations/ValidationHelper.cs#L76

Reply:
Ok will check this when merging :)